oaknational/oak-ai-lesson-assistant
# feat: remove feedback message from header

## Description

- Remove feedback form link from aila header

## Issue(s)

Fixes AI-442

## How to test

1. Go to https://oak-ai-lesson-assistant-apsbaxp1z.vercel-preview.thenational.academy
2. Go to /aila
3. You shouldn't see the feedback link in the header

## Screenshots

How it used to look (delete if n/a):
![CleanShot 2024-08-29 at 16 48 43@2x](https://github.com/user-attachments/assets/96f1cdd1-dfbf-4232-930e-79f6bd3c8506)

How it should now look:
![CleanShot 2024-08-29 at 16 52 55@2x](https://github.com/user-attachments/assets/c50abb9b-aae1-4b80-89f3-a97125628650)

## Checklist
